sdk/tables/azure-data-tables/src/table_clients.cpp

```diff
@@ -920,16 +758,16 @@ Azure::Response<Models::SubmitTransactionResult> TableClient::SubmitTransaction(
 {
   auto url = m_url;
   url.AppendPath("$batch");
-  std::string batchId = "batch_" + Azure::Core::Uuid::CreateUuid().ToString();
+  std::string batchIdUuid = "batch_" + Azure::Core::Uuid::CreateUuid().ToString();
   std::string changesetId = "changeset_" + Azure::Core::Uuid::CreateUuid().ToString();
```

Comment:
Name choice is a bit strange to me.
1. Uuid is Id, so Id is sufficient?
2. changeset is also Id, yet it is named just changesetId, and not changesetIdUuid.
3. If we want Uuid, can we just name them batchUuid and changesetUuid? Although, I think I'd prefer changesetId and batchId.